var/spack/repos/builtin/packages/cbtf-argonavis/package.py

```diff
@@ -57,14 +57,14 @@ class CbtfArgonavis(Package):
     version('1.6', branch='master',
             git='https://github.com/OpenSpeedShop/cbtf-argonavis.git')
 
-    depends_on("cmake@3.0.2", type='build')
+    depends_on("cmake@3.0.2:", type='build')
     depends_on("boost@1.50.0:")
+    # depends_on("boost@1.53.0")
```

Comment:
I will take that out.    I use 1_53_0 on my system to match installed components.